### PR TITLE
Feat/highlight new rls assistant

### PR DESCRIPTION
**Pull request by saltcod**
screenshot-2024-01-29-at-21 13 12

Highlight the new RLS Assistant from the old one

### CODE REVIEW

File: apps/studio/components/interfaces/Auth/Policies/PolicySelection.tsx [hunk 1]
```diff
@@ -58,6 +72,33 @@ const PolicySelection = ({
             onClick={onViewEditor}
           />
         </div>
+        <div className="bg-surface-200 rounded-md p-4 flex items-center  gap-2 text-sm">
```
Comment (MildTomato) [inline apps/studio/components/interfaces/Auth/Policies/PolicySelection.tsx:75]: we don't have a component for this?

File: apps/studio/components/interfaces/Auth/Policies/PolicySelection.tsx [hunk 2]
```diff
@@ -58,6 +72,33 @@ const PolicySelection = ({
...
+          <Info size={16} strokeWidth={2} />
```
Comment (MildTomato) [inline apps/studio/components/interfaces/Auth/Policies/PolicySelection.tsx:76]: We use hero icon for Info, Warning and Destructive icons

Comment (joshenlim) [reply apps/studio/components/interfaces/Auth/Policies/PolicySelection.tsx:76]: jfyi icons are here - https://github.com/supabase/supabase/blob/feat/highlight-new-rls-assistant/packages/ui/src/components/Admonition/index.tsx#L21

they aren't exported - will bring them out

File: apps/studio/components/interfaces/Auth/Policies/PolicySelection.tsx (hunk 1 above)
Comment (joshenlim) [reply apps/studio/components/interfaces/Auth/Policies/PolicySelection.tsx]: opted for alert shadcn component instead

**Review by alaister — APPROVED** [review]
LGTM!

**Pull request merged**